### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by zli82016**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+* **New Resource:** `google_vertex_ai_feature_online_store_featureview` ([#6821](https://github.com/hashicorp/terraform-provider-google-beta/pull/6821))
+* **New Resource:** `google_vmwareengine_external_access_rule` ([#6811](https://github.com/hashicorp/terraform-provider-google-beta/pull/6811))
+
+IMPROVEMENTS:
+* compute: added `md5_authentication_key` field to `google_compute_router_peer` resource ([#6815](https://github.com/hashicorp/terraform-provider-google-beta/pull/6815))
+* compute: added in-place update support to `params.resource_manager_tags` field in `google_compute_instance` resource ([#6828](https://github.com/hashicorp/terraform-provider-google-beta/pull/6828))
+* compute: added in-place update support to `description` field in `google_compute_instance` resource ([#6804](https://github.com/hashicorp/terraform-provider-google-beta/pull/6804))
+* gkehub: added `policycontroller` field to `google_gke_hub_feature_membership` resource. Users can now configure a Policy Controller for individual memberships in the fleet. ([#6813](https://github.com/hashicorp/terraform-provider-google-beta/pull/6813))
```
Comment (ScottSuarez) [inline CHANGELOG.md:16]: `Users can now configure a Policy Controller for individual memberships in the fleet`

Not sure what this part means. If it's just a reiteration that the field has been added we can likely remove it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+* servicenetworking: added field `deletion_policy` to `google_service_networking_connection` ([#6830](https://github.com/hashicorp/terraform-provider-google-beta/pull/6830))
+* sql: set `replica_configuration`, `ca_cert`, and `server_ca_cert` fields to be sensitive in `google_sql_instance` and `google_sql_ssl_cert` resources ([#6823](https://github.com/hashicorp/terraform-provider-google-beta/pull/6823))
+
+BUG FIXES:
+* bigquery: fixed perma-diff of `encryption_configuration` when API returns an empty object on `google_bigquery_table` resource ([#6817](https://github.com/hashicorp/terraform-provider-google-beta/pull/6817))
+* compute: fixed an issue where it would wait indefinitely for the resource to become stable when `wait_for_instances` is set to true before deleting on `google_compute_instance_group_manager` and `google_compute_region_instance_group_manager` resources ([#6829](https://github.com/hashicorp/terraform-provider-google-beta/pull/6829))
```
Comment (ScottSuarez) [inline CHANGELOG.md:25]:
```suggestion
* compute: fixed an issue where the provider would `wait_for_instances` if set before deleting on `google_compute_instance_group_manager` and `google_compute_region_instance_group_manager` resources ([#6829](https://github.com/hashicorp/terraform-provider-google-beta/pull/6829))
```

optional

**Review by ScottSuarez — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (zli82016) [reply CHANGELOG.md]: It is just a reiteration. Removed.

File: CHANGELOG.md (hunk 2 above)
Comment (zli82016) [reply CHANGELOG.md]: Done.

**Pull request merged**